index.src.html

```diff
@@ -691,16 +691,31 @@ <h3 id="user-mediation">
     Explicit user mediation
   </h3>
 
-  If a feature has privacy or security impacts that are endemic to the feature
-  itself, then one valid strategy for exposing it to the web is to require user
-  mediation before granting an origin access. For instance, [[GEOLOCATION-API]]
-  reveals a user's location, and wouldn't be particularly useful if it didn't;
+  If the security or privacy risk of a feature cannot otherwise be mitigated in
+  a specification, optionally allowing an implementer to prompt a user may
+  provide the best security/privacy possible.  If the specification does not
+  allow for the implementer to prompt, it may result in divergence
+  implementations by different user agents as some user agents choose to
+  implement more privacy-preserving version.
+
+  It is possible that the risk of a feature cannot be mitigated because the
+  risk is endemic to the feature itself.  For instance, [[ GEOLOCATION-API ]] 
+  reveals a user’s location, and wouldn’t be particularly useful if it didn’t;
   user agents generally gate access to the feature on a permission prompt which
-  the user may choose to accept.
-
-  Designing such prompts is difficult. Choosers are good. Walls of text are bad.
-
-  ISSUE: Bring in some of felt@'s ideas here.
+  the user may choose to accept.  This risk is also present and should be
+  accounted for in features that expose personal data or identifiers.
+
+  Designing such prompts is difficult as is determining the duration and timing
```

Comment:
It will be unclear for the reader what we mean by 'timing'. Do we really need this?

Reply:
Revised with https://github.com/w3cping/security-questionnaire/commit/db29182d06f131153f9134f2d7d25341f6b87b5f.